src/locales/ja/battle.ts

```diff
@@ -39,121 +39,121 @@ export const battle: SimpleTranslationEntries = {
+  "battlerTagsSeededLapseShed": "{{pokemonNameWithAffix}}は ヘドロえきを　すいとった！",
+  "battlerTagsNightmareOnAdd": "{{pokemonNameWithAffix}}は あくむを　みはじめた！",
+  "battlerTagsNightmareOnOverlap": "{{pokemonNameWithAffix}}は すでに　うなされている！",
+  "battlerTagsNightmareLapse": "{{pokemonNameWithAffix}}は あくむに　うなされている！",
+  "battlerTagsEncoreOnAdd": "{{pokemonNameWithAffix}}は アンコールをうけた！",
+  "battlerTagsEncoreOnRemove": "{{pokemonNameWithAffix}}の アンコールじょうたいが　とけた！",
+  "battlerTagsHelpingHandOnAdd": "{{pokemonNameWithAffix}}は {{pokemonName}}を\nてだすけする　たいせいに　はいった!",
+  "battlerTagsIngrainLapse": "{{pokemonNameWithAffix}}は　ねから\n ようぶんを　すいとった！",
+  "battlerTagsIngrainOnTrap": "{{pokemonNameWithAffix}}は　ねを　はった！",
+  "battlerTagsAquaRingOnAdd": "{{pokemonNameWithAffix}}は みずのリングを　まとった！",
+  "battlerTagsAquaRingLapse": "{{pokemonName}}は {{moveName}}で\nたいりょくを　かいふくした！",
+  "battlerTagsDrowsyOnAdd": "{{pokemonNameWithAffix}} の ねむけを　さそった！",
+  "battlerTagsDamagingTrapLapse": "{{pokemonNameWithAffix}}は　{{moveName}}の　ダメージを　うけた!",
+  "battlerTagsBindOnTrap": "{{pokemonNameWithAffix}}は　{{sourcePokemonName}}に しめつけられた!",
+  "battlerTagsWrapOnTrap": "{{pokemonNameWithAffix}}は {{sourcePokemonName}}に　まきつかれた！",
+  "battlerTagsVortexOnTrap": "{{pokemonNameWithAffix}}は うずの　なかに　とじこめられた！",
+  "battlerTagsClampOnTrap": "{{pokemonName}}は　{{sourcePokemonNameWithAffix}}の\nからに　はさまれた！",
+  "battlerTagsSandTombOnTrap": "{{pokemonNameWithAffix}}は　{{moveName}}に　とらわれた！",
+  "battlerTagsMagmaStormOnTrap": "{{pokemonNameWithAffix}}は マグマの\n　うずに　とじこめられた！",
+  "battlerTagsSnapTrapOnTrap": "{{pokemonNameWithAffix}}は　トラバサミに　とらわれた！",
+  "battlerTagsThunderCageOnTrap": "{{sourcePokemonNameWithAffix}}は {{pokemonNameWithAffix}}に　とじこめられた!",
+  "battlerTagsInfestationOnTrap": "{{pokemonNameWithAffix}}は　{{sourcePokemonNameWithAffix}}に　まとわりつかれた!",
+  "battlerTagsProtectedOnAdd": "{{pokemonNameWithAffix}}は\nまもりの　たいせいに　はいった！",
+  "battlerTagsProtectedLapse": "{{pokemonNameWithAffix}}は\nこうげきから　みを　まもった！",
+  "battlerTagsEnduringOnAdd": "{{pokemonNameWithAffix}}は\nこらえる　たいせいに　はいった！",
+  "battlerTagsEnduringLapse": "{{pokemonNameWithAffix}}は\nこうげきを　こらえた！",
+  "battlerTagsSturdyLapse": "{{pokemonNameWithAffix}}は\nこうげきを　こらえた！",
+  "battlerTagsPerishSongLapse": "{{pokemonNameWithAffix}}の ほろびのカウントが {{turnCount}}になった！",
+  "battlerTagsCenterOfAttentionOnAdd": "{{pokemonNameWithAffix}}は ちゅうもくの　まとになった！",
+  "battlerTagsTruantLapse": "{{pokemonNameWithAffix}}は なまけている！",
+  "battlerTagsSlowStartOnAdd": "{{pokemonNameWithAffix}}は ちょうしが　あがらない！",
+  "battlerTagsSlowStartOnRemove": "{{pokemonNameWithAffix}}は ちょうしを　とりもどした！",
+  "battlerTagsHighestStatBoostOnAdd": "{{pokemonNameWithAffix}}の {{statName}}が\nあがっている　じょうたいに　なった！",
+  "battlerTagsHighestStatBoostOnRemove": "{{pokemonNameWithAffix}}の　{{abilityName}}の　こうかが なくなった！",
+  "battlerTagsMagnetRisenOnAdd": "{{pokemonNameWithAffix}}は でんじりょくで　うかびあがった!",
+  "battlerTagsMagnetRisenOnRemove": "{{pokemonNameWithAffix}}は でんじりょくが　なくなった!",	
```

Comment:
```suggestion
  "battlerTagsMagnetRisenOnRemove": "{{pokemonNameWithAffix}}は でんじりょくが　なくなった!",
```
remove trailing space